### ISSUE TITLE
Show ornament previews in socket details sheet

### ISSUE THREAD

**Issue by bhollis** [DestinyItemManager/DIM#9780]
It'd be nice to see the full rendered preview of ornaments.

**Comment by Kuukuna93** [comment]
Got functional solution for this but to double check before putting up a PR, is this somewhat close the goal?

![image](https://github.com/DestinyItemManager/DIM/assets/20775789/39d5cebf-fc72-4e66-a870-3017159efb11)

**Comment by bhollis** [comment]
Yup, that's a good start. The tricky part of this task is designing where the preview should go in the UI. Ideally it's down with the rest of the socket preview, but doesn't take up too much space. There's also the mobile UI to consider.
